README.md

```diff
@@ -1,30 +1,80 @@
-# Brzeczyk
-A compiler implemented by TCS UJ students at "Compilers" course in the 2022/23 winter semester.
+# Brzęczyk
 
-## Building
-Project can be built using `gradle`. Installation of `gradle` is not required. To build, you can run
-```bash
-./gradlew build
-```
+[ˈbʒɛ̃n͇ʧ̑ɨk]
 
-## Testing
-Tests can be run with
-```bash
-./test_all.sh
-```
+A simple programming language with very *polished* syntax.
+
+Implemented by students at Theoretical Computer Science Department of Jagiellonian University
+as the project for *Compilers* course in the 2022/23 winter semester.
+
+The language does not really have anything out of ordinary and it's mostly C-like, with a few somewhat interesting (compared to C) features:
```

Comment:
Don't the below features count as something out of the ordinary? I think we can be a bit prouder :wink: 
```suggestion
The language itself is C-like, with some exotic features:
```

Reply:
I wouldn't really call them exotic, compared to modern programming languages.

Reply:
Removed the "nothing out of ordinary" part.